lonboard/geopandas.py

```diff
@@ -0,0 +1,323 @@
+                continuous.
+            elevation : Name of column on the dataframe used to extrude each geometry or
+                an array-like in the same order as observations. Defaults to None.
+            elevation_scale : Constant scaler multiplied by elevation value.
+            alpha : Alpha (opacity) parameter in the range (0,1) passed to
+                mapclassify.util.get_color_array.
+            layer_kwargs : Additional keyword arguments passed to lonboard.viz layer
+                arguments (either polygon_kwargs, scatterplot_kwargs, or path_kwargs,
+                depending on input geometry type).
+            map_kwargs : Additional keyword arguments passed to lonboard.viz map_kwargs.
+            classification_kwds : Additional keyword arguments passed to
+                `mapclassify.classify`.
+            nan_color : Color used to shade NaN observations formatted as an RGBA list.
+                Defaults to [255, 255, 255, 255]. If no alpha channel is passed it is
+                assumed to be 255.
+            color : single or array of colors passed to Layer.get_fill_color
+                or a lonboard.basemap object, or a string to a maplibre style basemap.
+            vmin : Minimum value for color mapping.
+            vmax : Maximum value for color mapping.
+            wireframe : Whether to use wireframe styling in deckgl.
+            tiles : Either a known string {"CartoDB Positron",
+                "CartoDB Positron No Label", "CartoDB Darkmatter",
+                "CartoDB Darkmatter No Label", "CartoDB Voyager",
+                "CartoDB Voyager No Label"}
+            highlight : Whether to highlight each feature on mouseover (passed to
+                lonboard.Layer's auto_highlight). Defaults to False.
+            m: An existing Map object to plot onto.
+
+        Returns:
+        lonboard.Map
+            a lonboard map with geodataframe included as a Layer object.
+
+        """
+        return _dexplore(
+            self._obj,
+            column=column,
+            cmap=cmap,
+            scheme=scheme,
+            k=k,
+            categorical=categorical,
+            elevation=elevation,
+            elevation_scale=elevation_scale,
+            alpha=alpha,
+            layer_kwargs=layer_kwargs,
+            map_kwargs=map_kwargs,
+            classification_kwds=classification_kwds,
+            nan_color=nan_color,
+            color=color,
+            vmin=vmin,
+            vmax=vmax,
+            wireframe=wireframe,
+            tiles=tiles,
+            highlight=highlight,
+            m=m,
+        )
+
+
+def _dexplore(  # noqa: C901, PLR0912, PLR0913, PLR0915
+    gdf,  # noqa: ANN001
+    *,
+    column,  # noqa: ANN001
+    cmap,  # noqa: ANN001
+    scheme,  # noqa: ANN001
+    k,  # noqa: ANN001
+    categorical,  # noqa: ANN001
+    elevation,  # noqa: ANN001
+    elevation_scale,  # noqa: ANN001
+    alpha,  # noqa: ANN001
+    layer_kwargs,  # noqa: ANN001
```

Comment:
It looks like `layer_kwargs` is something you're adding that doesn't exist in the upstream `GeoDataFrame.explore`. In that case, you should follow the same behavior as in `viz` and use the same parameters:

https://github.com/developmentseed/lonboard/blob/c6c320957511e87ea29db8c65e08c4de5230ec16/lonboard/_viz.py#L92-L95

This will also fix your type hinting below:
<img width="819" alt="image" src="https://github.com/user-attachments/assets/5decece2-fdc2-406a-bad2-3e1bc5b09f78" />